README.md

```diff
@@ -28,4 +73,23 @@ with tracer.start_span('foo'):
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
\ No newline at end of file
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Release Schedule
+
+OpenTelemetry Python is under active development. Our goal is to release an
+_alpha_ version of the library at the end of September 2019. This release isn't
+guaranteed to conform to a specific version of the specification, and is likely
+to include backwards-incompatible changes in later releases.
+
+| Component                   | Version | Target Date       |
+| --------------------------- | ------- | ----------------- |
+| Tracing API                 | Alpha   | September 30 2019 |
+| Tracing SDK                 | Alpha   | September 30 2019 |
+| Metrics API                 | Alpha   | September 30 2019 |
+| Metrics SDK                 | Alpha   | September 30 2019 |
+| Jaeger Trace Exporter       | Alpha   | Unknown           |
+| Prometheus Metrics Exporter | Alpha   | Unknown           |
+| Context Propagation         | Alpha   | September 30 2019 |
+| OpenTracing Bridge          | Alpha   | Unknown           |
+| OpenCensus Bridge           | Alpha   | Unknown           |
```

Comment:
is it worth covering propagation/formatters supported? We have B3 header support now, and I believe @a-feld is working on tracestate.

Reply:
That's why I think that the bridges are underspecified. A minimal bridge IMHO only needs to support instrumenting code, i.e. is completely independent of concrete propagators/formatters. But of course, a more complete compatibility would also imply support for all propagators and exporters that OC supports, including OC's own protocol (100% compat would even need to support plugging existing OC exporters into opentelemetry-python, but I don't think we are aiming for that). Which reminds me that the OC bridge will be somewhat trickier than the OT bridge because since there is no API layer, we will probably need to occupy the same package names that OC does to support migration without source modification?

Reply:
These topics mostly come from @tedsuo's release doc: https://docs.google.com/spreadsheets/d/1FHt3uJgQWN-UWbgV4uMIPUr36JiJCfZ7ryQbHfurjKA/edit#gid=0. There's a checklist section at the bottom that includes the Jaeger and Prometheus exporters, but I didn't include them here. If we want to commit to including them in the release we can add them.

I didn't include either bridge, but we may want to include at least the OpenTracing bridge here too and leave OC for a later release.